docs/ui.md

```diff
@@ -1,7 +1,10 @@
 # Kubernetes UI Instructions
 
 ## Kubernetes User Interface
-Kubernetes has an extensible user interface with default functionality that describes the current cluster. See the [README](../www/README.md) in the www directory for more information.
+Kubernetes has an extensible user interface with default functionality that describes the current cluster. 
```

Comment:
Sorry, but can we change this? Users don't care if the UI is "extensible", that's for the dev doc. How about:

Kubernetes has a web-based user interface that displays the current cluster state graphically.